### DISCUSSION TITLE
What tdzcnhfb.com is?

### DISCUSSION THREAD

**Discussion by shiedltv61** [badmojr/1Hosts#1011] [Q&A]
I just see that 1host max blocks it. Since yesterday I've received many of them, so I decided to block them because I didn't find any information on google

**Comment by andy-page-developer** [comment]
Gotten 500 requests from this in the last 48 hrs. It's blocked but I'm intrigued to know what's calling it.

**Reply by b4dmojr** [reply]
I don't know. That's something for you to share with us!

**Reply by andy-page-developer** [reply]
Ha, if I ever find out I will. There is not a lot of info on the net about it at all. I'll keeping Dig(ging)

**Reply by andy-page-developer** [reply]
It seems to be related to ExpressVPN. I'm going to email them and find out what it's doing.

It looks like it's a call to home.
